velox/type/tz/TimeZoneMap.cpp

```diff
@@ -25,15 +25,20 @@
 
 namespace facebook::velox::tz {
 
+using TTimeZoneDatabase = std::vector<std::unique_ptr<TimeZone>>;
```

Comment:
@mbasmanova I ended up merging both PR into one; this is the final state as I mentioned in the previous review.